nixos-generate

```diff
@@ -80,7 +80,7 @@ while [[ $# -gt 0 ]]; do
       shift
       ;;
     --option)
-      nix_build_args+=("$1" "$2" "$3")
+      nix_build_args+=("$2" "$3")
```

Comment:
I don't understand. Don't we need to keep `--option` here?

i.e. if we have is equal to `--option foo bar` than we want `nix_build_args+=("--option" "foo" "bar")`

Reply:
Is there an option like `--option` for the nix-build command?

I was trying to use `--option` to pass `--builders "ssh://root@192.168.1.10"` to `nixos-generate`. Because of the current behavior, nix-build gets `--option --builders "ssh://root@192.168.1.10"` and as a result builds are not sent to the right builder.